skema/skema-rs/comment_extraction/src/bin/extract_comments.rs

```diff
@@ -37,15 +39,23 @@ fn main() {
         if let Some(path) = args.output {
             write(path, comments).expect("Unable to write to file!");
         } else {
-            println!("{comments}");
+            println!("{:?}", comments);
```

Comment:
@jastier  - Can you change this line back to the way it was earlier? We want to print actual JSON, not the debug output.